topology/sof/pipe-low-latency-playback.m4

```diff
@@ -32,21 +32,21 @@ include(`mixercontrol.m4')
 #
 # Volume Mixer control with max value of 32
 C_CONTROLMIXER(PCM PCM_ID Playback Volume, PIPELINE_ID,
-	CONTROLMIXER_OPS(volsw, 256 binds the mixer control to volume get/put handlers, 256, 256),
+	CONTROLMIXER_OPS(volsw, 1 binds the mixer control to volume get/put handlers, 1, 1),
 	CONTROLMIXER_MAX(, 32),
 	false,
 	CONTROLMIXER_TLV(TLV 32 steps from -90dB to +6dB for 3dB, vtlv_m90s3),
 	Channel register and shift for Front Left/Right,
-	LIST(KCONTROL_CHANNEL(FL, 0, 0), KCONTROL_CHANNEL(FR, 0, 1)))
+	LIST(KCONTROL_CHANNEL_ARRAY(PIPELINE_CHANNELS, 0)))
 
 # Volume Mixer control with max value of 32
 C_CONTROLMIXER(Master Playback Volume, PIPELINE_ID,
-	CONTROLMIXER_OPS(volsw, 256 binds the mixer control to volume get/put handlers, 256, 256),
+	CONTROLMIXER_OPS(volsw, 1 binds the mixer control to volume get/put handlers, 1, 1),
```

Comment:
here too, why change to 1?

Reply:
see my above comments and thesofproject/linux#39